src/components/ServiceList.jsx

```diff
@@ -30,32 +65,76 @@ const ServiceList = () => {
     );
   }
 
-  return isLoading ? (
-    <p>Loading Baltimore County services...</p>
-  ) : (
-    <div>
-      <div style={flex}>
-        <i
-          className="fas fa-star dg_popular_star"
-          aria-hidden="true"
-          style={MostPopularServiceIconStyles}
-        ></i>
-        <p style={legendText}> - Indicates a Most Popular Service</p>
-      </div>
-      <div className="row">
-        <FilterList
-          items={serviceItems}
-          renderItem={props => (
-            <div
-              key={props.name.replace(/\s/, "-")}
-              className="d-flex col-lg-3 col-md-6 col-sm-6"
-            >
-              <ServiceIconLink {...props} />
+  let searchItemFound =
```

Comment:
I think there was a comment here, but somehow it got clobbered.

One the ternary can be simplified to 

```js
const searchItemFound = !isFiltering && filteredItems.length > 0;
```

note not tested.

and then maybe a variable name of something along the lines of `hasFilteredResults` or something along those lines?

Reply:
updated